pkg/controller/proxyconfig/validation.go

```diff
@@ -93,27 +108,30 @@ func (r *ReconcileProxyConfig) ValidateProxyConfig(proxyConfig *configv1.ProxySp
 // validateTrustedCA validates that trustedCA is a valid ConfigMap
 // reference and that the ConfigMap contains a valid trust bundle,
 // returning the byte slices of the certificate data from the
-// validated trustedCA and system trust bundles.
-func (r *ReconcileProxyConfig) validateTrustedCA(trustedCA string) ([]byte, []byte, error) {
+// validated trustedCA and system trust bundles. If trustedCA is nil,
+// "trusted-ca-bundle" is used.
```

Comment:
hm.  If trustedCA is nil, shouldn't we be using the system-bundle (and then updating trusted-ca-bundle to be the system bundle, if validation passes)?

Reply:
@bparees yes, if `trustedCA` in proxy spec is nil, then the system bundle is [generated](https://github.com/openshift/cluster-network-operator/blob/master/pkg/controller/proxyconfig/controller.go#L138-L148) and will be used. `validateTrustedCA()` requires a string which represents the value of the `openshift-config-managed/trusted-ca-bundle` configmap. I think I can drop the string parameter from this func, but I would like to do that in a follow-on PR b/c I am currently rebuilding my cluster and will be unable to test. I will update the comment to provide a better explanation.